source/algorithms.tex

```diff
@@ -2509,12 +2510,12 @@
   }
 
   template<class BidirectionalIterator, class Predicate>
-    constexpr BidirectionalIterator stable_partition(BidirectionalIterator first,
+    constexpr BidirectionalIterator stable_partition(BidirectionalIterator first,   // hosted
                                                      BidirectionalIterator last,
                                                      Predicate pred);
   template<class ExecutionPolicy, class BidirectionalIterator, class Predicate>
-    BidirectionalIterator stable_partition(ExecutionPolicy&& exec,  // see \ref{algorithms.parallel.overloads}
-                                           BidirectionalIterator first,
+    BidirectionalIterator stable_partition(ExecutionPolicy&& exec,                  // hosted,
```

Comment:
This line is fine, but you need to add `// hosted` annotations to the ranges overloads of `stable_partition`.

Reply:
Fixing.